src/encoded/tests/data/inserts/user.json

```diff
@@ -964,5 +964,28 @@
             "community"
         ],
         "uuid": "75659c39-eaea-47b7-ba26-92e9ff183e6c"
+    },
```

Comment:
Flagging for possible merge conflict after merging, #2404 

Merge the other PR first.